app/graphql/graphql_devise/mutations/resend_confirmation.rb

```diff
@@ -0,0 +1,30 @@
+module GraphqlDevise
+  module Mutations
+    class ResendConfirmation < Base
+      argument :email,        String, required: true, prepare: ->(email, _) { email.downcase }
+      argument :redirect_url, String, required: true
+      
+      field :success, Boolean, null: false
```

Comment:
As there is no way for this to be `false` I think it is good to remove it. GraphQL kind of assumes success if the `error` key of the response is empty. That's why we raise execution  errors that add error messages to that key. Message field is fine

Reply:
Got it. That makes sense and was something I was considering removing. Looks like we've got a consensus on that.